app/assets/stylesheets/pages/watchlist.scss

```diff
@@ -0,0 +1,80 @@
+body.page-watchlist{
```

Comment:
I would rather not create a new dir, and instead make change this to `components/corner-ribbon.scss`. Also remove the root page-watchlist class.